assets/js/type-defs/index.d.ts

```diff
@@ -0,0 +1,2 @@
+export * from './cart-response';
```

Comment:
Is there a reason to write `.d.ts` (declaration) files? I would avoid writing these files and just use `.ts`. There's no requirement to include code in `.ts` files.

Reply:
That's my lack of familiarity with TypeScript showing :). I still don't fully understand the difference between declaration files and regular ts files. I thought this would be a good way to signal this file only contains non compiled code (thus only types, interfaces and declarations).

Reply:
done in b1f1cc0 and 3e5b949.